contracts/deploy/foreign/optimism.js

```
@@ -44,10 +44,11 @@ async function deployForeignProxy({ deploy, from, parameters, homeNetworkName, h
   const { numberOfJurors, foreignBridge, variant } = parameters;
   const metaEvidence = getMetaEvidenceCID(homeNetworkName);
   const arbitratorExtraData = encodeExtraData(courts.oracle, numberOfJurors);
+  const chainId = await getChainId();
```

ESLint

[error] 'getChainId' is not defined.

(no-undef)